src/domains/sign/cosmos_issuer.tf

```diff
@@ -80,9 +80,9 @@ module "cosmosdb_sql_container_issuer-issuers-by-vat-number" {
   default_ttl = var.io_sign_database_issuer.issuers.ttl
 }
 
-module "cosmosdb_sql_container_issuer-issuers-ip-whitelist" {
+module "cosmosdb_sql_container_issuer-issuers-whitelist" {
   source              = "git::https://github.com/pagopa/terraform-azurerm-v3.git//cosmosdb_sql_container?ref=v4.1.11"
-  name                = "issuers-ip-whitelist"
+  name                = "issuers-whitelist"
```

Comment:
⚠️ Beware that renaming a collection implies destroying the one with the old name and creating a new one with the changed name.

Please make sure to backup the data before applying